### PR TITLE
[MINI-3450]Update the Message API to support an optional message

### PR DESCRIPTION
**Pull request by munir-rakuten**
# Description
Update the Link Message API to support an optional message to the user for the contact picker capability

Screen Shot 2021-07-09 at 18 49 21

## Links
MINI-3450

# Checklist
- [x] I have read the [contributing guidelines](../.github/CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](../.github/CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `./gradlew check` without errors

### CODE REVIEW

**Review by khairul-alam-licon — APPROVED** [review]
LGTM

File: testapp/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -10,4 +10,5 @@
     <color name="window_top_bar">#F8F5F5</color>
     <color name="qr_error_title">#333333</color>
     <color name="link_blue">#358ED7</color>
+    <color name="colorPink">#EBCAC9</color>
```
Comment (Climbatize) [inline testapp/src/main/res/values/colors.xml:13]: I suggest a Rakuten Crimson color with 20% of opacity
```suggestion
    <color name="colorPink">#33BE0000</color>
```

**Review by Climbatize — APPROVED** [review]
LGTM, just read my suggestion ;)

**Review by corycaywood — APPROVED** [review]
👍

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +14,7 @@
 - **Fix:** Prevent exception during calling `onError` asynchronously in `getUniqueId`.
 - **Update:** Supported "Update codebase" functionality for preview mode.
 - **Update:** Included technical improvement in custom permission and manifest information caching.
+- **Update:** Update `MessageToContact` to support an optional message to the user for the contact picker capability.
```
Comment (corycaywood) [inline CHANGELOG.md:17]: @munir-rakuten @khairul-alam-licon This item was added to the wrong version - it should be under 3.X.X. We need to remember to move this before the release.

Comment (khairul-alam-licon) [reply CHANGELOG.md:17]: Yeah, I'll add a separate PR for this.

Comment (khairul-alam-licon) [reply CHANGELOG.md:17]: Yeah, I have addressed this issue in https://github.com/rakutentech/android-miniapp/pull/346
Thank you for noticing.

**Pull request merged**